.github/pull_request_template.md

```diff
@@ -1,8 +1,10 @@
+Attached to issue: https://github.com/DataBiosphere/azul/issues/
```

Comment:
I've seen several different formats for this. WHat does the official ZH doc say?

```suggestion
Connected to issue: https://github.com/DataBiosphere/azul/issues/
```

Reply:
https://github.com/DataBiosphere/azul/issues/3219#issuecomment-877552166, but let's discuss in PL.

Reply:
@hannes-ucsc suggests using *just* the link to keep it simple/minimize confusion.